src/components/App/App.js

```diff
@@ -1,15 +1,35 @@
 import React from 'react';
-import logo from '../../logo.svg';
-import { UserPage } from '../../containers/UserPage/UserPage';
-
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route,
+    Redirect
+} from "react-router-dom";
+import {UserPage} from '../../containers/UserPage/UserPage';
+import {PostDetailsPage} from "../../containers/PostDetailsPage/PostDetailsPage";
+import {NotFoundPage} from "../../containers/NotFoundPage/NotFoundPage";
 import './App.css';
 
 function App() {
-  return (
-    <div className="App">
-      <UserPage />
-    </div>
-  );
+    return (
+        <Router>
+            <Switch>
+                <Redirect exact from="/" to="/users"/>
+                <Route exact path="/users">
+                    <div className="App">
```

Comment:
I guess this div is useless and can be deleted. And if you want to stylize the page in the way .App is stylized you can copy styles to the UserPage.scss